maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestController.java

```diff
@@ -50,6 +50,19 @@ public ResponseEntity<List<ConcorContributionResponse>> concorContributionFiles(
         return ResponseEntity.ok(contributionResponses);
     }
 
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @StandardApiResponse
+    @GetMapping(value = "/concor-contribution-file/{concorContributionId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Get the Concor Contribution ID and associated XML for given concorContributionId")
+    public ResponseEntity<ConcorContributionResponse> concorContributionFile(
+        @PathVariable final Integer concorContributionId
+    ) {
+        log.info("Get Concor Contribution ID and associated XML for concorContributionId {}", concorContributionId);
+        ConcorContributionResponse contributionResponse = concorContributionsService.getConcorContributionFile(concorContributionId);
+
+        return ResponseEntity.ok(contributionResponse);
```

Comment:
Ideally we would return a `404 Not Found` if the contribution file doesn't exist.